tests/post_training/requirements.txt

```diff
@@ -7,7 +7,7 @@ onnx==1.13.1
 onnxruntime==1.14.1
 pytest
 pytest-cov
-openvino-dev==2023.0.1
+openvino-dev==2023.1.0.dev20230811
```

Comment:
Why was it changed? Do we have any proof that this package is not buggy?

Reply:
2023.1.0.dev20230811 have some bug that was fixed on current release branch.
But with 2023.0.1 mo.convert_model does not support quantized models.
Until dump new openvino package it's the best option.